### DISCUSSION TITLE
Getting repeat index

### DISCUSSION THREAD

**Discussion by timathom** [Jinntec/Fore#209] [Q&A] [answered]
Is it possible to get the index of the currently selected repeat item in Fore? In [this demo](https://jinntec.github.io/Fore/demo/repeat-nested.html), I see the `index()` function is used, but it's not working in the form. I see that the `@index` attribute dynamically changes in the DOM, but the displayed values in the form are always "1;1".

![Screenshot 2023-04-21 at 9 01 08 AM](https://user-images.githubusercontent.com/5334524/233642610-735f849a-af1b-4b6e-bfa1-4d390ef67e2c.png)

**Comment by JoernT** [comment, marked as answer]
Yes, index() is the right function - getting the idref of the repeat

The example is somewhat broken though it should work - need to have a look.

However in insert.html you can see the index() function working.
